php/blocks/controls/class-image.php

```diff
@@ -46,15 +53,30 @@ public function register_settings() {
 				'sanitize' => 'sanitize_text_field',
 			)
 		);
-		$this->settings[] = new Control_Setting(
-			array(
-				'name'     => 'default',
```

Comment:
You probably saw this, but this PR removes the default value setting:

<img width="1117" alt="image-field" src="https://user-images.githubusercontent.com/4063887/55522135-ef0ccd00-5640-11e9-9ff0-b302348861d0.png">

This is because this control now stores the integer image ID, not the URL string.

The change below is a backward-compatibility break. It won't honor default image URLs that were entered before.